src/main/java/net/openhft/chronicle/core/onoes/ThreadLocalisedExceptionHandler.java

```diff
@@ -24,7 +24,15 @@ public class ThreadLocalisedExceptionHandler implements ExceptionHandler {
     private ExceptionHandler defaultHandler;
     private ThreadLocal<ExceptionHandler> handlerTL;
 
+    /**
+     * @deprecated use {@link ThreadLocalisedExceptionHandler(ExceptionHandler)} instead (to be removed in .24)
+     */
+    @Deprecated
```

Comment:
`binary-compatibility-enforcer` made me do it

Reply:
hehe. minor but please use x.24 for easier greppage